src/qt/rpcconsole.cpp

```diff
@@ -1118,6 +1119,9 @@ void RPCConsole::updateNodeDetail(const CNodeCombinedStats *stats)
     ui->peerDirection->setText(stats->nodeStats.fInbound ? tr("Inbound") : tr("Outbound"));
     ui->peerHeight->setText(QString("%1").arg(QString::number(stats->nodeStats.nStartingHeight)));
     ui->peerWhitelisted->setText(stats->nodeStats.m_legacyWhitelisted ? tr("Yes") : tr("No"));
+    if (stats->nodeStats.m_mapped_as != 0) {
+        ui->peerMappedAS->setText(QString("%1").arg(QString::number(stats->nodeStats.m_mapped_as)));
+    }
```

Comment:
I think you need to set something here in the case of `0` as well.

Reply:
Thanks -- are you thinking of something like this?
```diff
     if (stats->nodeStats.m_mapped_as != 0) {
         ui->peerMappedAS->setText(QString("%1").arg(QString::number(stats->nodeStats.m_mapped_as)));
+    else
+        ui->peerMappedAS->setText(tr("N/A"));
     }
```
It looks like in the case of `0` it already falls back to the "N/A" value in debugwindow.ui::1521
```
                 <property name="text">
                  <string>N/A</string>
                 </property>
```
similar to`peerPingWait` in rpcconsole.cpp::1114 a few lines above, which displays "Ping Wait: N/A" in the GUI example images above. I'm not 100% sure -- further guidance here welcome.

Reply:
Yes, you have to reset to default N/A label.

Reply:
It doesn't appear to be needed from my testing on Debian and macOS, which defaults to the value set in debugwindow.ui::1521 -- nevertheless, deferring to your experience here. Done.